minari/utils.py

```diff
@@ -216,6 +380,17 @@ def create_dataset_from_buffers(
             "`author_email` is set to None. For longevity purposes it is highly recommended to provide an author email, or some other obvious contact information.",
             UserWarning,
         )
+    if minari_version is None:
+        warnings.warn(
+            f"`minari_version` is set to None. The compatible dataset version specifier for Minari will be automatically fixed to the installed version {__version__}.",
+            UserWarning,
+        )
+        minari_version = f"=={__version__}"
```

Comment:
maybe we want to support all the version with the first two numbers equal, i.e. for minari `0.3.1` we want `0.3.x` (?)

Reply:
I agree. This makes more sense to be the default.